### PR TITLE
Brushup EC book 5 NPCs part 2

### PR DESCRIPTION
**Pull request by roxim5**

### CODE REVIEW

File: packs/data/extinction-curse-bestiary.db/urdefhan-dominator.json [hunk 1]
```diff
@@ -1076,7 +1078,7 @@
                 },
                 "area": {
                     "areaType": "burst",
-                    "value": 20
+                    "value": "20"
```
Comment (simonward) [inline packs/data/extinction-curse-bestiary.db/urdefhan-dominator.json:1081]: Stringified

File: packs/data/extinction-curse-bestiary.db/urdefhan-dominator.json [hunk 2]
```diff
@@ -2234,7 +2237,7 @@
                 },
                 "area": {
                     "areaType": "line",
-                    "value": 30
+                    "value": "30"
```
Comment (simonward) [inline packs/data/extinction-curse-bestiary.db/urdefhan-dominator.json:2240]: Stringified

File: packs/data/extinction-curse-bestiary.db/ammut.json [hunk 3]
```diff
@@ -773,7 +787,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The ammut exhales a blast of flame that deals [[/r {18d6}[fire]]]{18d6 fire damage} to all creatures in a 30 foot cone (@Check[type:reflex|dc:37|basic:true] save). The ammut can't use their Breath Weapon again for [[/br 1d4]] rounds.</p>"
+                    "value": "<p>The ammut exhales a blast of flame that deals [[/r {18d6}[fire]]]{18d6 fire damage} to all creatures in a @Template[type:cone|distance:30]{30-foot cone} (@Check[type:reflex|dc:37|basic:true] save). The ammut can't use their Breath Weapon again for [[/br 1d4]] rounds.</p>"
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/ammut.json:790]:
```suggestion
                    "value": "<p>The ammut exhales a blast of flame that deals [[/r {18d6}[fire]]]{18d6 fire damage} to all creatures in a @Template[type:cone|distance:30]{30-foot cone} (@Check[type:reflex|dc:37|basic:true] save). The ammut can't use their Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]] rounds.</p>"
```

File: packs/data/extinction-curse-bestiary.db/kharostan.json [hunk 4]
```diff
@@ -338,14 +332,14 @@
...
-            "_id": "UaXs7wrcxQyf2m3B",
+            "_id": "nycgOFC3jC4CfnmG",
             "flags": {
                 "core": {
                     "sourceId": "Compendium.pf2e.spells-srd.TTwOKGqmZeKSyNMH"
                 }
             },
             "img": "systems/pf2e/icons/spells/feather-fall.webp",
-            "name": "Feather Fall (At Will) (Self Only)",
+            "name": "Feather Fall (At Will)(Self Only)",
```
Comment (SpartanCPA) [inline packs/data/extinction-curse-bestiary.db/kharostan.json:342]:
```suggestion
            "name": "Feather Fall (At Will) (Self Only)",
```

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**